### PR TITLE
Feature/roc 2435 add change links to claimant check and send

### PR DESCRIPTION
**Pull request by jacmurphy**
### JIRA link ###
https://tools.hmcts.net/jira/browse/ROC-2435

### Change description ###
Added change options to claim - check and send

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/features/claim/views/check-and-send.njk [hunk 1]
```diff
@@ -14,48 +14,58 @@
...
     {% endif %}
 
     {{ row("Claim fee:", claimAmountTotal.feeAmount | numeral) }}
-    {{ row('Total amount you’re claiming:',  claimAmountTotal.totalAmount | numeral) }}
+    {{ row('Total amount you’re claiming:',  claimAmountTotal.totalAmount | numeral, paths.totalPage.uri) }}
     {{ tableEnd() }}
 
     {{ tableStart("claimDetails", "Claim details") }}
-    {{ row('Why you’re owed the money:', draftClaim.reason.reason ) }}
+    {{ row('Why you’re owed the money:', draftClaim.reason.reason , paths.reasonPage.uri ) }}
```
Comment (timja-kainos) [inline src/main/features/claim/views/check-and-send.njk]: extra space is after reason currently

File: src/main/features/claim/views/check-and-send.njk [hunk 2]
```diff
@@ -14,48 +14,58 @@
...
-        {{ row("Date interest is claimed from:", t( 'Date you submit the claim' )) }}
+        {{ row("Date interest is claimed from:", t( 'Date you submit the claim' ), paths.interestDatePage.uri) }}
       {% endif %}
       {% if (claimAmountTotal.interestAmount) %}
-        {{ row("Interest:", claimAmountTotal.interestAmount | numeral) }}
+        {{ row("Interest:", claimAmountTotal.interestAmount | numeral , paths.interestPage.uri) }}
       {% endif %}
     {% else %}
-      {{ row("Interest rate:", t('I don’t want to claim interest')) }}
+      {{ row("Interest rate:", t('I don’t want to claim interest') , paths.interestPage.uri )}}
```
Comment (timja-kainos) [inline src/main/features/claim/views/check-and-send.njk]: there's an extra space after `) ,`

File: src/main/features/claim/views/check-and-send.njk [hunk 3]
```diff
@@ -14,48 +14,58 @@
...
       {% if payAtSubmission === false %}
-        {{ row("Date interest is claimed from:", draftClaim.interestDate.date.asString() | date ) }}
+        {{ row("Date interest is claimed from:", draftClaim.interestDate.date.asString() | date , paths.interestDatePage.uri) }}
       {% else %}
-        {{ row("Date interest is claimed from:", t( 'Date you submit the claim' )) }}
+        {{ row("Date interest is claimed from:", t( 'Date you submit the claim' ), paths.interestDatePage.uri) }}
       {% endif %}
       {% if (claimAmountTotal.interestAmount) %}
-        {{ row("Interest:", claimAmountTotal.interestAmount | numeral) }}
+        {{ row("Interest:", claimAmountTotal.interestAmount | numeral , paths.interestPage.uri) }}
```
Comment (timja-kainos) [inline src/main/features/claim/views/check-and-send.njk]: same here

File: src/main/features/claim/views/check-and-send.njk [hunk 4]
```diff
@@ -14,48 +14,58 @@
...
+    {{ row("Claim amount:", claimAmountTotal.claimAmount | numeral , paths.amountPage.uri )}}
 
     {% if (interestClaimed === true) %}
       {% if (draftClaim.interest.rate) %}
-        {{ row("Interest rate:", draftClaim.interest.rate | numeral("0,0.00") + "\% \(" + draftClaim.interest.type + "\)" ) }}
+        {{ row("Interest rate:", draftClaim.interest.rate | numeral("0,0.00") + "\% \(" + draftClaim.interest.type + "\)" , paths.interestPage.uri) }}
       {% endif %}
       {% if payAtSubmission === false %}
-        {{ row("Date interest is claimed from:", draftClaim.interestDate.date.asString() | date ) }}
+        {{ row("Date interest is claimed from:", draftClaim.interestDate.date.asString() | date , paths.interestDatePage.uri) }}
```
Comment (timja-kainos) [inline src/main/features/claim/views/check-and-send.njk]: and here

File: src/main/features/claim/views/check-and-send.njk [hunk 5]
```diff
@@ -14,48 +14,58 @@
...
+        )
     }}
-    {{ row("Email:", draftClaim.defendant.email.address) }}
+
+    {{ row("Email:", draftClaim.defendant.email.address , paths.defendantEmailPage.uri) }}
     {{ tableEnd() }}
 
     {{ tableStart("claimAmount", "Claim amount") }}
-    {{ row("Claim amount:", claimAmountTotal.claimAmount | numeral) }}
+    {{ row("Claim amount:", claimAmountTotal.claimAmount | numeral , paths.amountPage.uri )}}
```
Comment (timja-kainos) [inline src/main/features/claim/views/check-and-send.njk]: and here

File: src/main/features/claim/views/check-and-send.njk [hunk 6]
```diff
@@ -14,48 +14,58 @@
...
+            showCorrespondenceAddressSection = false,
+              changeLinks = {
+              name: paths.defendantIndividualDetailsPage.uri,
+              address: paths.defendantIndividualDetailsPage.uri
+           }
+        )
     }}
-    {{ row("Email:", draftClaim.defendant.email.address) }}
+
+    {{ row("Email:", draftClaim.defendant.email.address , paths.defendantEmailPage.uri) }}
```
Comment (timja-kainos) [inline src/main/features/claim/views/check-and-send.njk]: and here

File: src/main/features/claim/views/check-and-send.njk (hunk 1 above)
Comment (jacmurphy) [reply src/main/features/claim/views/check-and-send.njk]: Updated

File: src/main/features/claim/views/check-and-send.njk (hunk 2 above)
Comment (jacmurphy) [reply src/main/features/claim/views/check-and-send.njk]: Updated

File: src/main/features/claim/views/check-and-send.njk (hunk 3 above)
Comment (jacmurphy) [reply src/main/features/claim/views/check-and-send.njk]: Updated

File: src/main/features/claim/views/check-and-send.njk (hunk 4 above)
Comment (jacmurphy) [reply src/main/features/claim/views/check-and-send.njk]: Updated

File: src/main/features/claim/views/check-and-send.njk (hunk 5 above)
Comment (jacmurphy) [reply src/main/features/claim/views/check-and-send.njk]: Updated

File: src/main/features/claim/views/check-and-send.njk (hunk 6 above)
Comment (jacmurphy) [reply src/main/features/claim/views/check-and-send.njk]: Updated

Comment (jacmurphy) [reply src/main/features/claim/views/check-and-send.njk]: All spaces have been updated with your suggested changes

**Comment by jacmurphy** [comment]
Getting and error on Jenkins Build 
```

[cmc-citizen-frontend_PR-185-IASW6IZE6T54PXUWNNC4QWWNNQL6J35NFF6VAY2CIERKERBRQZNQ] Running shell script
+ yarn install
yarn install v1.3.2
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
error An unexpected error occurred: "https://registry.yarnpkg.com/@hmcts/cmc-common-frontend/-/cmc-common-frontend-1.16.0.tgz: connect ECONNREFUSED 104.16.61.173:443".
info If you think this is a bug, please open a bug report with the information provided in "/home/jenkins/slave/workspace/cmc-citizen-frontend_PR-185-IASW6IZE6T54PXUWNNC4QWWNNQL6J35NFF6VAY2CIERKERBRQZNQ/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
error https://registry.yarnpkg.com/ast-types/-/ast-types-0.9.14.tgz: connect ECONNREFUSED 104.16.61.173:443
error https://registry.yarnpkg.com/escodegen/-/escodegen-1.9.0.tgz: connect ECONNREFUSED 104.16.61.173:443
script returned exit code 1

```

**Comment by rlewan** [comment]
@jacmurphy just restart a job, it's an erratic error we started to get recently, ops are working to resolve this.

**Pull request merged**